src/Php85/Resources/stubs/Uri/Rfc3986Uri.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Uri;
+
+if (\PHP_VERSION_ID < 80500) {
+    class Rfc3986Uri extends \Symfony\Polyfill\Php85\Uri\Rfc3986Uri
```

Comment:
The classes in core don't have a parent. Shall we alias the class instead?

Reply:
I can see `class Rfc3986Uri extends Uri` in the RFC. Do I miss your point?

Reply:
Yes, but `Symfony\Polyfill\Php85\Uri\Rfc3986Uri` does not appear in the original inheritance chain. What is the point of the Symfony namespaced classes anyway? Do we need them?

Reply:
Having all the logic being in internal namespaced classes allows to test the logic of those classes even on versions where the polyfill is not applied because the native class exist. That's why all our polyfill classes have such internal parent class.